cmd/buildah/run.go

```diff
@@ -85,9 +89,9 @@ func runCmd(c *cli.Context) error {
 		Runtime:  runtime,
 		Args:     flags,
 	}
-	runerr := builder.Run(c.Args(), options)
+	runerr := builder.Run(args, options)
 	if runerr != nil {
-		logrus.Debugf("error running %v in container: %v", c.Args(), runerr)
+		logrus.Debugf("error running %v in container: %v", args, runerr)
```

Comment:
Should this this still be c.Args?  args might not have the name in the args[0], is that important for this debug output?